src/constants/abacus.ts

```diff
@@ -179,8 +179,10 @@ export type ValidationError = Abacus.exchange.dydx.abacus.output.input.Validatio
 export const TradeInputErrorAction = Abacus.exchange.dydx.abacus.output.input.ErrorAction;
 export type AbacusOrderTypes = Abacus.exchange.dydx.abacus.output.input.OrderType;
 export type AbacusOrderSides = Abacus.exchange.dydx.abacus.output.input.OrderSide;
+export type AbacusOrderTimeInForces = Abacus.exchange.dydx.abacus.output.input.OrderTimeInForce;
```

Comment:
duplicate?

Reply:
hmm I don't think so / this is the `type` vs `const` on line 185 (was following the pattern of `AbacusOrderTypes`/`AbacusOrderType` and `AbacusOrderSides`/`AbacusOrderSide`